main/res/values/strings.xml

```diff
@@ -747,23 +747,23 @@
+    <string name="init_dbonsdcard_note">You may store the database of c:geo either system internally or in a folder on your user storage. Having it on the user storage may free some memory, but you may lose a bit of performance and c:geo will not work if the storage is not accessible or you remove the database file.</string>
     <string name="init_dbonsdcard">On user storage</string>
     <string name="init_dbmove_dbmove">Moving Database</string>
     <string name="init_dbmove_running">Moving Database</string>
     <string name="init_dbmove_success">Successfully moved the database.</string>
     <string name="init_dbmove_failed">Failed to move database</string>
-    <string name="init_datadirmove_datadirmove">Moving Geocache data directory</string>
-    <string name="init_datadirmove_running">Moving Geocache data directory</string>
-    <string name="init_datadirmove_success">Successfully moved Geocache data directory.</string>
-    <string name="init_datadirmove_failed">Moving the Geocache data directory partly or completely failed. Please check old and new directories and move by hand if necessary.</string>
+    <string name="init_datadirmove_datadirmove">Moving Geocache data folder</string>
+    <string name="init_datadirmove_running">Moving Geocache data folder</string>
+    <string name="init_datadirmove_success">Successfully moved Geocache data folder.</string>
+    <string name="init_datadirmove_failed">Moving the Geocache data folder partly or completely failed. Please check old and new folders and move by hand if necessary.</string>
     <string name="init_plain_logs">Plain Logs</string>
     <string name="init_summary_plain_logs">Display log entries without colors</string>
     <string name="init_use_native_ua">Android browser</string>
     <string name="init_summary_use_native_ua">Identify as Android browser. Solves login problems when using certain network providers.</string>
-    <string name="init_rendertheme_folder">Map Themes Directory</string>
+    <string name="init_rendertheme_folder">Map Themes Folder</string>
     <string name="init_maintenance">Maintenance</string>
-    <string name="init_maintenance_directories_note">c:geo stores images, log images and other files related to a cache in a separate directory. In some cases (like importing/exporting the database) this directory may contain outdated files, which can be deleted here.</string>
+    <string name="init_maintenance_directories_note">c:geo stores images, log images and other files related to a cache in a separate folder. In some cases (like importing/exporting the database) this folder may contain outdated files, which can be deleted here.</string>
```

Comment:
How about: `c:geo stores images and other files related to a cache in a separate folder. In some cases (e.g. when importing/exporting the database) this folder may contain obsolete files that can be deleted here.`